### PR TITLE
Logo added and headerprofile UI fix

### PR DESCRIPTION
**Pull request by SidHarsh25**

### CODE REVIEW

File: elemental/module/membership/library/class-loginhandler.php [hunk 1]
```diff
@@ -629,8 +629,8 @@ private function company_header_logo(array $companyDetails = null): ?string
 		ob_start();
 ?>
 
-		<a href="<?php echo esc_url($store_url); ?>" class="elemental-host-link">
```
Comment (ccfredbaum) [inline elemental/module/membership/library/class-loginhandler.php]: please run wordpress check style going forward - but for today I will do it AFTER PR is merged on server.
esc_url( $store .... note the space between $ and (

File: elemental/assets/css/elemental.css [hunk 2]
```diff
@@ -893,4 +893,10 @@ table td {
 
 .ui-tabs .ui-tabs-nav {
     padding: 0px ! important;
+}
+
+.elementor-44807 .elementor-element.elementor-element-81c2990>.elementor-container {
```
Comment (ccfredbaum) [inline elemental/assets/css/elemental.css:898]: Do not use elementor style sheets - rather create a class in the object and use it in our own CSS> Elementor sheets can change in document names etc and will not work across multiple servers as they are done by post name.

Comment (ccfredbaum) [reply elemental/assets/css/elemental.css:898]: (or you can apply the style elsewhere)

File: elemental/module/membership/library/class-loginhandler.php [hunk 3]
```diff
@@ -629,8 +629,8 @@ private function company_header_logo(array $companyDetails = null): ?string
...
-			<div class="elemental-primary-nav-img one" style="width:100%; background-image: url(<?php echo esc_url($store_logo); ?> )"></div>
+		<a href="<?php echo esc_url($store_url); ?>" class="elemental-host-link logoCompany">
+			<div class="elemental-primary-nav-img oneLogo" style="width:100%; height:49px; background-image: url(<?php echo esc_url($store_logo); ?> )"></div>
```
Comment (ccfredbaum) [inline elemental/module/membership/library/class-loginhandler.php:633]: using camel case for style- should use dashes and all lower case.

File: elemental/assets/css/elemental.css [hunk 4]
```diff
@@ -310,7 +309,7 @@
 }
 
 .elemental-background-item {
```
Comment (ccfredbaum) [inline elemental/assets/css/elemental.css:311]: I have fixed a few of these in main already - please rebase before sending through

File: elemental/assets/css/elemental.css [hunk 5]
```diff
@@ -812,6 +811,7 @@ a {
     /* background-position: top; */
     background-repeat: no-repeat;
     background-size: cover ! important;
+    background: url(https://coadjute.app/wp-content/uploads/2022/08/Backgr_.jpg);
```
Comment (ccfredbaum) [inline elemental/assets/css/elemental.css:814]: same as above re rebase

File: elemental/module/membership/views/membership/view-edituser.php [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+ *  @package  module/membership/views/view-edituser.php
+ */
+
+
+return function (
+    object $current_user
+): string {
+    ob_start();
+
+    if (user_can($current_user, 'administrator')) {
```
Comment (ccfredbaum) [inline elemental/module/membership/views/membership/view-edituser.php:15]: This is not right - as only WordPress Admins can use this form - and tenant admin accounts have to use it - a better check is for a Tenant account user type, or currently "store_vendor" role

File: elemental/module/membership/views/membership/view-edituser.php [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+                            <h4 id="edit_notifyuser" style="    text-align: center;color:#388015"> </h4>
+                        </div>
+                    </div>
+
+                    <div class="row erow">
+                        <div class="col-xs-12" id="demoContainer">
+                            <form id="registrationForm" action="elemental_editcompany_ajax" method="post" class="form-horizontal fv-form fv-form-bootstrap ajax" novalidate="novalidate">
+
+                                <div class="form-group">
+                                    <label class="col-xs-3 control-label">Email address</label>
```
Comment (ccfredbaum) [inline elemental/module/membership/views/membership/view-edituser.php:37]: No translations - please use esc_html__( 'Email Address', 'elementalplugin' );  --- applies to all fields where text is output

File: elemental/module/membership/views/membership/view-edituser.php [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+                            </form>
+                        </div>
+                    </div>
+
+
+                </div><!-- #content -->
+            </div><!-- #primary -->
+    <?php
+    } else {
+        echo 'Login as admin ';
```
Comment (ccfredbaum) [inline elemental/module/membership/views/membership/view-edituser.php:78]: same as above - tenant admins need to use this not just wordpress admins

**Review by ccfredbaum — CHANGES_REQUESTED** [review]

**Review by ccfredbaum — CHANGES_REQUESTED** [review]
Please change your IDE to code style WordPress - you are checking in code in PEAR format and its undoing all of the standard formatting. I can fix by running phpcbf - but ideally code standard should be done before any merge - loginhandler is a particular issue

**Comment by SerhiiStarovoitov** [comment]
> Please change your IDE to code style WordPress - you are checking in code in PEAR format and its undoing all of the standard formatting. I can fix by running phpcbf - but ideally code standard should be done before any merge - loginhandler is a particular issue

I will run it on the server for now.

**Pull request merged**